frontends/common/resolveReferences/resolveReferences.cpp

```diff
@@ -16,74 +16,57 @@ limitations under the License.
 
 #include "resolveReferences.h"
 
-#include <sstream>
-
-#include <boost/range/adaptor/reversed.hpp>
```

Comment:
Always good to see boost dependencies being removed :).

Reply:
We just replace with out own boost ;)